data/json/monsters/mi-go.json

```diff
@@ -83,7 +83,12 @@
     "special_attacks": [
       [ "PARROT", 800 ],
       { "id": "scratch", "damage_max_instance": [ { "damage_type": "cut", "amount": 23, "armor_multiplier": 0.8 } ] },
-      { "type": "spell", "spell_data": { "id": "mi-go_slaver_beam", "min_level": 3 }, "cooldown": 100 }
+      {
+        "type": "spell",
+        "spell_data": { "id": "mi-go_slaver_beam", "min_level": 3 },
+        "cooldown": 100,
+        "monster_message": "The mi-go's numerous antennae sync their movement only for a moment."
```

Comment:
Minor editorial change. This is purely down to personal preference on my part. Feel free to ignore:

`The mi-go's numerous antennae sync their movement for a brief moment.`

Reply:
Oh yeah no, I'd say I am really bad at descriptions and text in genral, anything is welcome